src/components/Sidebar.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {List, Box, ListItemAvatar, Avatar, ListItemText, ListItem, ListItemIcon, Typography} from '@material-ui/core';
+import { AiFillHome } from "react-icons/ai";
+import { FaChessBoard } from "react-icons/fa";
+import { GiChessRook } from "react-icons/gi";
+import { FaBook } from "react-icons/fa";
+import { FaGlobe } from "react-icons/fa";
+
+
+
+
+class Sidebar extends Component {
```

Comment:
Was this component actually used?

Reply:
This was the original design. I ended up changing it.